Sources/Datadog/OpenTracing/OTTracer.swift

```diff
@@ -87,4 +105,24 @@ public extension OTTracer {
             startTime: startTime
         )
     }
+
+    /// Start a new root span with the given operation name.
+    /// - Parameters:
+    ///   - operationName: the operation name for the newly-started span
+    ///   - tags:          a set of tag keys and values per `OTSpan#setTag:value:`, or `nil` to start with
+    ///                    an empty tag map
+    ///   - startTime:     an explicitly specified start timestamp for the `OTSpan`, or `nil` to use the
+    ///                    current walltime
+    /// - returns:         a valid Span instance; it is the caller's responsibility to call `finish()`.
+    func startRootSpan(
+        operationName: String,
+        tags: [String: Encodable]? = nil,
+        startTime: Date? = nil
+    ) -> OTSpan {
+        return self.startRootSpan(
+            operationName: operationName,
+            tags: tags,
+            startTime: startTime
+        )
```

Comment:
this code goes into infinite loop
do we need this extended method?

Reply:
As discussed, it goes infinite only in case of `NoOpTracer`. We will fix it 👍.